package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "nutrinova",
```

Comment:
Looks like an extra lock file got generated.

Reply:
I must have tried to npm i outside of the ui project